### PR TITLE
De-betaing the TFE v5 docs

### PR DESCRIPTION
**Pull request by nfagerlund**
This is probably ready to merge once we deal with any issues y'all spot during review. 

Here's a shot of part of the sidebar: 

image

### CODE REVIEW

File: content/source/docs/enterprise/before-installing/cluster-architecture.html.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+A Terraform Enterprise cluster relies on a load balancer to direct traffic to application instances.
+
+## Architecture Diagrams
+
+The following diagrams represent the architectures used on the three supported cloud platforms:
+
+### AWS
+
+![AWS architecture diagram. Includes a load balancer, three primary instances, and multiple secondary instances in multiple zones, whach are part of an auto-scaling group.](https://github.com/hashicorp/terraform-aws-terraform-enterprise/blob/v0.0.1-beta/assets/aws_diagram.jpg?raw=true)
```
Comment (amy-hashi) [inline content/source/docs/enterprise/before-installing/cluster-architecture.html.md]: Can we update the diagram link to point to master? That way if we update those in the future we won't have to also update the website links.

File: content/source/docs/enterprise/before-installing/cluster-architecture.html.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+### GCP
+
+![Google Cloud Platform architecture diagram. Includes a load balancer, three primary instances, and multiple secondary instances in multiple zones, which are part of a managed instance group.](https://github.com/hashicorp/terraform-google-terraform-enterprise/blob/v0.0.1-beta/assets/gcp_diagram.jpg?raw=true)
+
+### Azure
+
+![Azure architecture diagram. Includes a load balancer, three primary instances, and multiple secondary instances, which are part of a scaleset. Also includes an adjacent key vault for TLS.](../install/assets/azure_diagram.png)
```
Comment (amy-hashi) [inline content/source/docs/enterprise/before-installing/cluster-architecture.html.md]: And let's point this one to the diagram on github as opposed to a local one - https://github.com/hashicorp/terraform-azurerm-terraform-enterprise/blob/master/assets/azure_diagram.png?raw=true

File: content/source/docs/enterprise/before-installing/cluster-architecture.html.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+## Data Storage
+
+Clustered deployment is designed for use with external data services. These include a PostgreSQL database, a blob storage service, and optionally an external Vault cluster.
```
Comment (amy-hashi) [inline content/source/docs/enterprise/before-installing/cluster-architecture.html.md]: We'd like to remove references to external vault in this setup - there's no support for it in the modules.

File: content/source/docs/enterprise/before-installing/index.html.md [hunk 4]
```diff
@@ -5,31 +5,33 @@ page_title: "Pre-Install Checklist - Before Installing - Terraform Enterprise"
...
-3. **Data storage:** Depending on your operational mode, prepare data storage services or a block storage device.
-4. **Linux instance:** Prepare a Linux instance for Terraform Enterprise. This might require additional configuration or software installation, depending on the OS and your operational requirements.
+1. **Choose a deployment method:** Decide whether to perform a clustered deployment of Terraform Enterprise (using a HashiCorp-provided Terraform module) or use the installer to deploy individual instances.
+2. **Choose an operational mode:** Decide how Terraform Enterprise should store its data. This is affected by your choice of deployment method.
+3. **Credentials:** Ensure you have a Terraform Enterprise license and a TLS certificate for Terraform Enterprise to use.
+4. **Data storage:** Depending on your operational mode, prepare data storage services or a block storage device.
+5. **Linux instance:** Prepare a Linux machine image (if clustering) or a running Linux instance (if deploying individually) for Terraform Enterprise. This might require additional configuration or software installation, depending on the OS and your operational requirements.
```
Comment (amy-hashi) [inline content/source/docs/enterprise/before-installing/index.html.md]: Should we point out preparing an image is optional? They can use a custom OS image, or use the cloud-provided ones we've got as default.

File: content/source/docs/enterprise/before-installing/index.html.md [hunk 5]
```diff
@@ -73,13 +90,18 @@ Make sure your data storage services or device meet Terraform Enterprise's requi
...
+Terraform Enterprise runs on Linux instances. The source of these instances depends on your deployment method:
+
+- **Clustered deployment:** Terraform automatically provisions all instances for Terraform Enterprise. The machine image and the instance type are configurable:
+    - By default, the module uses an official Ubuntu image; you can override this with any image that meets the [software requirements below](#software-requirements).
+    - The default instance type depends on the cloud you deploy to; see the module documentation for details. You can override this, and can optionally specify separate image types for primary and secondary instances.
+- **Individual deployment:** You must prepare a running Linux instance for Terraform Enterprise before running the installer. You will start and manage this instance like any other server.
 
-The Terraform Enterprise installer currently supports the following
-operating systems:
+In either case, note that Terraform Enterprise currently supports running under the following operating systems:
```
Comment (amy-hashi) [inline content/source/docs/enterprise/before-installing/index.html.md]: Clustering only supports Ubuntu, and RHEL 7.2-7.6 at the moment. (RHEL 7.7 is specifically unsupported in the current version of replicated.)

File: content/source/docs/enterprise/before-installing/index.html.md (hunk 4 above)
Comment (nfagerlund) [reply content/source/docs/enterprise/before-installing/index.html.md]: Mm, good call.

File: content/source/docs/enterprise/before-installing/index.html.md (hunk 5 above)
Comment (nfagerlund) [reply content/source/docs/enterprise/before-installing/index.html.md]: 👍🏼 !

File: content/source/docs/enterprise/before-installing/cluster-architecture.html.md (hunk 3 above)
Comment (nfagerlund) [reply content/source/docs/enterprise/before-installing/cluster-architecture.html.md]: Oh, didn't know that. 👍🏼

File: content/source/docs/enterprise/before-installing/cluster-architecture.html.md (hunk 1 above)
Comment (nfagerlund) [reply content/source/docs/enterprise/before-installing/cluster-architecture.html.md]: lol, didn't even notice that. 👍🏼

File: content/source/docs/enterprise/before-installing/cluster-architecture.html.md (hunk 2 above)
Comment (nfagerlund) [reply content/source/docs/enterprise/before-installing/cluster-architecture.html.md]: 👍🏼

Comment (erindatkinson) [reply content/source/docs/enterprise/before-installing/cluster-architecture.html.md]: Although if we point it to a local copy then when GitHub goes down our site isn't affected.

**Comment by nfagerlund** [comment]
OK, I'm gonna just turn this into a non-draft now; there's always more to do, but I think this is in a state where it's a significant improvement that doesn't break anything.

File: content/redirects.txt [hunk 6]
```diff
@@ -366,3 +366,9 @@
 
 # API Token Documentation update
 /docs/cloud/users-teams-organizations/service-accounts.html         /docs/cloud/users-teams-organizations/api-tokens.html
+
+# Post-beta for TFE v5
+/docs/enterprise/beta/index.html        /docs/enterprise/beta/before-installing/index.html
```
Comment (glasner) [inline content/redirects.txt]: I *think* the target is wrong here?

Comment (nfagerlund) [reply content/redirects.txt]: YUP, thank you for catching

**Pull request merged**